recipes/rocksdict/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rocksdict" %}
+{% set version = "0.3.23" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Congyuwang/RocksDict/archive/v{{ version }}.tar.gz
+  sha256: 79441cd6668b337649027df4612382d1f28fd8cf5d41edc11e99cc8f3f5eb9cb
+
+build:
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  script_env:
+    - LIBCLANG_PATH=$BUILD_PREFIX/lib  # [not win]
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - clangdev
+    - clangxx
```

Comment:
Can you instead add a conda_build_config.yaml with
```
c_compiler:
  - clang
cxx_compiler:
  - clangxx
```

If that works, it's probably the more sustainable approach.

```suggestion
```

Reply:
let me try that thanks @xhochy

Reply:
@xhochy no luck; should I add `clangdev` back to build? Should I leave `conda_build_config` with clang in or out?

Reply:
Hmmm actually the error is something else 
```
  - nothing provides requested clang_linux-64 13.*
  - nothing provides requested clangxx_linux-64 13.*
```

Reply:
https://conda-forge.org/news/2024/04/30/clang-everywhere/ maybe I need to specify compiler version like it says here?